odoo/addons/base/res/res_company.py

```diff
@@ -331,11 +331,6 @@ def _onchange_rml_paper_format(self):
         else:
             self.rml_header = self._header_a4
```

Comment:
Maybe rml_paper_format could be removed as well and paperformat_id (from the report module) becomes the proper one? Not quite sure about that one /cc @sle-odoo @rco-odoo